azkaban-common/src/main/java/azkaban/constants/PropertyFilePaths.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package azkaban.constants;
+
+public class PropertyFilePaths {
```

Comment:
The context key doesn't seem to be path related.

What do you think about naming it something more generic like Server and put all server internal constants here?